hera_corr_cm/hera_corr_cm.py

```diff
@@ -355,32 +362,34 @@ def get_ant_status(self):
         stats = self._get_status_keys("snap")
         # For the conv_info dictionary below, the format is:
         #     key: name of the variable in the return dictionary from this method
-        #     tuple:  (redis key name, conversion method from redis to this method).
+        #     tuple:  (redis key name,
+        #              conversion method from redis to this method,
+        #              arg for conversion method).
         conv_info = {
-            'adc_mean': ('stream{$CH}_mean', float),
-            'adc_rms': ('stream{$CH}_rms', float),
-            'adc_power': ('stream{$CH}_power', float),
-            'pam_atten': ('pam{$PF}_atten_{$POL}', int),
-            'pam_power': ('pam{$PF}_power_{$POL}', float),
-            'pam_voltage': ('pam{$PF}_voltage', float),
-            'pam_current': ('pam{$PF}_current', float),
-            'eq_coeffs': ('stream{$CH}_eq_coeffs', json.loads),
-            'histogram': ('stream{$CH}_hist', json.loads),
-            'autocorrelation': ('stream{$CH}_autocorr', json.loads),
-            'fem_lna_power': ('fem{$PF}_lna_power_{$POL}', lambda x: (x == 'True')),
-            'pam_id': ('pam{$PF}_id', json.loads),
-            'fem_temp': ('fem{$PF}_temp', float),
-            'fem_voltage': ('fem{$PF}_voltage', float),
-            'fem_current': ('fem{$PF}_current', float),
-            'fem_pressure': ('fem{$PF}_pressure', float),
-            'fem_humidity': ('fem{$PF}_humidity', float),
-            'fem_id': ('fem{$PF}_id', json.loads),
-            'fem_switch': ('fem{$PF}_switch', str),
-            'fem_imu_theta': ('fem{$PF}_imu_theta', float),
-            'fem_imu_phi': ('fem{$PF}_imu_phi', float),
-            'timestamp': ('timestamp', dateutil.parser.parse),
-            'clip_count': ('eq_clip_count', int),
-            'fft_of': ('fft_overflow', lambda x: (x == 'True'))
+            'adc_mean': ('stream{$CH}_mean', float, None),
+            'adc_rms': ('stream{$CH}_rms', float, None),
+            'adc_power': ('stream{$CH}_power', float, None),
+            'pam_atten': ('pam{$PF}_atten_{$POL}', int, None),
+            'pam_power': ('pam{$PF}_power_{$POL}', float, None),
+            'pam_voltage': ('pam{$PF}_voltage', float, None),
+            'pam_current': ('pam{$PF}_current', float, None),
+            'eq_coeffs': ('stream{$CH}_eq_coeffs', np.frombuffer, float),
+            'histogram': ('stream{$CH}_hist', np.frombuffer, int),
+            'autocorrelation': ('stream{$CH}_autocorr', np.frombuffer, float),
```

Comment:
The ndarrays are converted using the tobytes() method with no arguments.  I'm happy to convert as needed.

My approach has been to work towards hera_mc by finalizing the interface out of hera_corr_f via hera_corr_cm, then update hera_mc.  Aaron has merged hera_corr_f into master.

Reply:
yeah the conversion too is simple and out on this end is a little more complicated because we need to know what the types were more strictly. I'm pretty sure the floats should be np.float32